lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -525,6 +510,25 @@ def reload(options = nil)
           self
         end
       end
+
+      private
+
+      def bitemporal_assign_initialize_value(valid_datetime, current_time)
```

Comment:
Can you change to keyword arguments?
`bitemporal_assign_initialize_value(valid_datetime: Time.current, current_time: Time.current)` is more readable than `bitemporal_assign_initialize_value(Time.current, Time.current)`

Reply:
Sure, but was it intended that the default value for `valid_datetime` is `Time.current`? The current implementation is that the argument is required.

Reply:
> The current implementation is that the argument is required.

Yes. Example implementation.

```ruby
def bitemporal_assign_initialize_value(valid_datetime:, current_time: Time.current)
  # ...
end
bitemporal_assign_initialize_value(valid_datetime: target_datetime, current_time: any_time_current)
```
Or maybe even `def bitemporal_assign_initialize_value(valid_datetime, current_time: Time.current)`.

Reply:
Fixed https://github.com/kufu/activerecord-bitemporal/pull/74/commits/32b4599b2b043cf7dd82db44247f0aa548ed8e79

Reply:
Thanks! :)